addons/account/migrations/13.0.1.1/pre-migration.py

```diff
@@ -146,36 +152,24 @@ def create_account_move_new_columns(env):
             )
 
 
-def fill_account_move_line_parent_state(env):
-    """Faster way"""
-    openupgrade.logged_query(
-        env.cr, """
-        ALTER TABLE account_move_line
-        ADD COLUMN parent_state varchar""",
-    )
-    openupgrade.logged_query(
-        env.cr, """
-        UPDATE account_move_line aml
-        SET parent_state = am.state
-        FROM account_move am
-        WHERE am.id = aml.move_id""",
-    )
-
-
-def fill_account_move_line_account_internal_type(env):
+def fill_account_move_line(env):
     """Faster way"""
     openupgrade.logged_query(
         env.cr, """
         ALTER TABLE account_move_line
+        ADD COLUMN parent_state varchar,
         ADD COLUMN account_internal_type varchar""",
     )
     openupgrade.logged_query(
         env.cr, """
         UPDATE account_move_line aml
-        SET account_internal_type = aat.type
-        FROM account_account aa
-        JOIN account_account_type aat ON aa.user_type_id = aat.id
-        WHERE aml.account_id = aa.id""",
+        SET parent_state = am.state
```

Comment:
account_internal_type?

Reply:
Thanks, error joining it in a single script smile :smile: